site/src/components/diagrams/Diagrams.module.css

```diff
@@ -17,8 +17,8 @@
 }
 
 .imageBorder {
-  border: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
-  padding: calc(var(--salt-spacing-200) - var(--salt-size-border));
+  border: var(--salt-size-fixed-100) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
+  padding: calc(var(--salt-spacing-200) - var(--salt-size-fixed-100));
```

Comment:
Couldn't we just making this `--salt-spacing-100` and remove the need for a calc or have I misunderstood the calc being done here?

Reply:
It's the make it an "inside" border which is why we have to do it so much in code :( 

The calculation removes 1px from --salt-spacing-200